integration/mediation-tests/tests-mediator-1/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/sequences/MessageProcessorWSATestSequence.xml

```diff
@@ -0,0 +1,40 @@
+<!--
+ ~  Copyright (c) 2017, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ ~
+ ~  WSO2 Inc. licenses this file to you under the Apache License,
+ ~  Version 2.0 (the "License"); you may not use this file except
+ ~  in compliance with the License.
+ ~  You may obtain a copy of the License at
+ ~
+ ~    http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~  Unless required by applicable law or agreed to in writing,
+ ~  software distributed under the License is distributed on an
+ ~  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~  KIND, either express or implied.  See the License for the
+ ~  specific language governing permissions and limitations
+ ~  under the License.
+ ~
+ -->
+
+<sequence xmlns="http://ws.apache.org/ns/synapse" name="MessageProcessorWSATestSequence" trace="disable">
```

Comment:
Do we need to have a separate sequence since it's only being used in one place?

Reply:
Removed the sequence and added the mediators inside the proxy itself.